clustered/server/entity/src/main/java/org/ehcache/clustered/server/ConcurrencyStrategies.java

```diff
@@ -25,6 +25,7 @@
 import org.terracotta.entity.ConcurrencyStrategy;
 
 import static java.util.Collections.singleton;
+import org.ehcache.clustered.server.internal.messages.EhcacheMessageTrackerCatchup;
```

Comment:
Could you move this up in to the regular imports (not with the statics).

Reply:
fixed